lib/generators/kubelogic/config_generator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
```

Comment:
I believe it should be lib/kubelogic/generators/

Reply:
In this case I got error like ```